### ISSUE TITLE
Error making patch release train

### ISSUE THREAD

**Issue by gavin-norman-sociomantic** [sociomantic-tsunami/neptune#72]
(This time pasting the complete output of the tool ;)
```
gavinnorman@labs-080:~/devel/swarm-tsunami:v4.1.x %=$ ../neptune/build/devel/bin/neptune-release 
We are on branch v4.1.x
Detected release v4.1.6
Actions to be done:
 * Creating annotated tag v4.1.6 (git tag -F- v4.1.6 v4.1.x)
 * Checkout v4.2.x locally (git checkout v4.2.x)
 * Merge v4.1.6 into v4.2.x (git merge -m "Merge tag v4.1.6 into v4.2.x" v4.1.6)
 * Creating annotated tag v4.2.1 (git tag -F- v4.2.1 v4.2.x)
 * Checkout v4.x.x locally (git checkout v4.x.x)
 * Merge v4.2.1 into v4.x.x (git merge -m "Merge tag v4.2.1 into v4.x.x" v4.2.1)
 * Checkout original branch v4.1.x (git checkout v4.1.x)

Would you like to continue? y/n: y

Creating annotated tag v4.1.6 ...
	git tag -F- v4.1.6 v4.1.x
	
Checkout v4.2.x locally ...
	git checkout v4.2.x
	Switched to branch 'v4.2.x'
Your branch is up-to-date with 'upstream/v4.2.x'.
Merge v4.1.6 into v4.2.x ...
	git merge -m "Merge tag v4.1.6 into v4.2.x" v4.1.6
	Merge made by the 'recursive' strategy.
 src/swarm/neo/client/mixins/AllNodesRequestCore.d | 11 +++++++++--
 src/swarm/neo/node/Connection.d                   |  2 --
 2 files changed, 9 insertions(+), 4 deletions(-)
Creating annotated tag v4.2.1 ...
	git tag -F- v4.2.1 v4.2.x
	
Checkout v4.x.x locally ...
	git checkout v4.x.x
	Switched to branch 'v4.x.x'
Your branch is up-to-date with 'upstream/v4.x.x'.
Merge v4.2.1 into v4.x.x ...
	git merge -m "Merge tag v4.2.1 into v4.x.x" v4.2.1
	Merge made by the 'recursive' strategy.
 src/swarm/neo/client/mixins/AllNodesRequestCore.d | 11 +++++++++--
 src/swarm/neo/node/Connection.d                   |  2 --
 2 files changed, 9 insertions(+), 4 deletions(-)
Checkout original branch v4.1.x ...
	git checkout v4.1.x
	Switched to branch 'v4.1.x'
Your branch is up-to-date with 'upstream/v4.1.x'.
Done! Some references should be pushed: ["v4.1.6", "v4.2.1", "v4.2.x"]
Would you like to push the modified references now? y/n: y
Some tags on github should be released: ["v4.1.6", "v4.2.1"]
Would you like to release those tags now? y/n: y
Creating v4.1.6 ...
object.Exception@/home/gavinnorman/devel/neptune/submodules/vibed/source/vibe/stream/openssl.d(279): Reading from TLS stream was unsuccessful with ret 0
----------------
/usr/include/dmd/phobos/std/exception.d:420 pure @safe void std.exception.bailOut!(Exception).bailOut(immutable(char)[], ulong, const(char[])) [0x84f04e]
/usr/include/dmd/phobos/std/exception.d:388 pure @safe bool std.exception.enforce!(Exception, bool).enforce(bool, lazy const(char)[], immutable(char)[], ulong) [0x84efca]
/home/gavinnorman/devel/neptune/submodules/vibed/source/vibe/stream/openssl.d:279 int vibe.stream.openssl.OpenSSLStream.checkSSLRet(int, immutable(char)[]) [0x8ad519]
/home/gavinnorman/devel/neptune/submodules/vibed/source/vibe/stream/openssl.d:206 void vibe.stream.openssl.OpenSSLStream.read(ubyte[]) [0x8acf65]
./submodules/vibed/source/vibe/stream/operations.d:355 void vibe.stream.operations.readUntilSmall!(vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender).readUntilSmall(vibe.core.stream.InputStream, ref vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender, const(ubyte[]), ulong) [0x827907]
./submodules/vibed/source/vibe/stream/operations.d:137 void vibe.stream.operations.readUntil!(vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender).readUntil(vibe.core.stream.InputStream, ref vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender, const(ubyte[]), ulong) [0x82773a]
./submodules/vibed/source/vibe/stream/operations.d:53 void vibe.stream.operations.readLine!(vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender).readLine(vibe.core.stream.InputStream, ref vibe.utils.array.AllocAppender!(ubyte[], ubyte).AllocAppender, ulong, immutable(char)[]) [0x8276dc]
./submodules/vibed/source/vibe/stream/operations.d:39 ubyte[] vibe.stream.operations.readLine!().readLine(vibe.core.stream.InputStream, ulong, immutable(char)[], vibe.utils.memory.Allocator) [0x827696]
./submodules/vibed/source/vibe/http/client.d:845 vibe.http.client.HTTPClientResponse vibe.http.client.HTTPClientResponse.__ctor(vibe.http.client.HTTPClient, bool, bool, vibe.utils.memory.Allocator, std.datetime.SysTime) [0x82f566]
./submodules/vibed/source/vibe/http/client.d:489 vibe.http.client.HTTPClientResponse vibe.http.client.HTTPClient.request(scope void delegate(vibe.http.client.HTTPClientRequest)) [0x82dad2]
./submodules/octod/src/octod/core.d:278 vibe.data.json.Json octod.core.HTTPConnection.post(immutable(char)[], vibe.data.json.Json) [0x84d042]
./src/release/github.d:90 void release.github.createRelease(ref octod.core.HTTPConnection, ref octod.api.repos.Repository, immutable(char)[], immutable(char)[], immutable(char)[]) [0x867c5c]
/home/gavinnorman/devel/neptune/src/release/main.d:138 _Dmain [0x75adb5]
??:? _D2rt6dmain211_d_run_mainUiPPaPUAAaZiZ6runAllMFZ9__lambda1MFNlZv [0x908d53]
??:? scope void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x908c83]
??:? scope void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).runAll() [0x908cfc]
??:? scope void rt.dmain2._d_run_main(int, char**, extern (C) int function(char[][])*).tryExec(scope void delegate()) [0x908c83]
??:? _d_run_main [0x908bf3]
??:? main [0x7607a5]
??:? __libc_start_main [0x7e7dd82f]
```

**Comment by gavin-norman-sociomantic** [comment]
The upstream v4.x.x had not been pushed.

**Comment by mathias-baumann-sociomantic** [comment]
I've stumbled over this one while testing as well, however only for other parts where I interact with the github api. The only solution I found was to simply retry the request. I guess I'll have to add some retry code to all api interactions.

**Comment by mathias-baumann-sociomantic** [comment]
In fact, I have fixed this already in 5cf9384dc6fc0f40251c12f1dc31e9a871366696

**Comment by mathias-baumann-sociomantic** [comment]
If *someone* would merge it at some point you would even benefit from it, too ;)
